lib/installation_user_settings.pm

```diff
@@ -29,10 +30,10 @@ sub type_password_and_verification {
 sub await_password_check {
     # PW too easy (cracklib)
     # bsc#937012 is resolved in > SLE 12, skip if VERSION=12
-    return if (is_sle('=12') && check_var('ARCH', 's390x'));
-    assert_screen 'inst-userpasswdtoosimple';
+    return if (is_sle('=12') && is_s390x);
+    my $timeout = (is_ppc64le) ? 90 : 30;
+    assert_screen 'inst-userpasswdtoosimple', $timeout;
```

Comment:
What about to set it to 90 for all without  arch check?

Reply:
Turns out it's a product bug: https://bugzilla.suse.com/show_bug.cgi?id=1158996